_data/tools.yml

```diff
@@ -756,6 +756,23 @@
       url: https://www.magnitude.com/drivers/trino-odbc-jdbc
     - urltext: Starburst ODBC driver
       url: https://docs.starburst.io/data-consumer/clients/odbc.html
+- name: OpenAPI
+  anchor: openapi
+  category: data-source
+  logo: /assets/images/logos/openapi.png
+  logosmall: /assets/images/logos/openapi-small.png
+  description: |
+    The OpenAPI Specification is a specification language for HTTP APIs that
+    provides a standardized means to define your API to others. You can quickly
+    discover how an API works, configure infrastructure, generate client code,
+    and create test cases for your APIs.
+
+    Trino has a OpenAPI connector for accessing APIs with an OpenAPI spec.
```

Comment:
Use any REST API that publishes an OpenAPI specification as a data source in Trino by configuring a catalog with the OpenAPI connector.